File: branches/Andean/Fantasy.Base/Windows/NotifyPropertyChangedObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace Fantasy.Windows
{
    public class NotifyPropertyChangedObject : INotifyPropertyChanged
    {
        public event PropertyChangedEventHandler PropertyChanged;

        protected virtual void OnPropertyChanged(string propertyName)
        {
            if (PropertyChanged != null)
            {
                this.PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
            }
        }
    }
}

File: AutoConverted/Fantasy.Base/Windows/NotifyPropertyChangedObject.java
package Fantasy.Windows;

public class NotifyPropertyChangedObject implements INotifyPropertyChanged
{
//C# TO JAVA CONVERTER TODO TASK: Events are not available in Java:
//	public event PropertyChangedEventHandler PropertyChanged;

	protected void OnPropertyChanged(String propertyName)
	{
		if (PropertyChanged != null)
		{
			this.PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
		}
	}
}